async-clipboard/demo.js

```diff
@@ -0,0 +1,55 @@
+const copy = document.querySelector('#copy');
+const paste = document.querySelector('#paste');
+const out = document.querySelector('#out');
+
+/** Write contents of the textarea to the clipboard when clicking "Copy" */
+copy.onclick = () => {
+  navigator.clipboard.writeText(out.value)
+    .then(result => {
+      ChromeSamples.log('Text copied.' + result);
```

Comment:
According to the [spec](https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/external/wpt/interfaces/clipboard-apis.idl?q=navigator+clipboard+writeText&sq=package:chromium&dr=C&l=1), there's nothing returned.

```js
navigator.clipboard.writeText(out.value)
    .then(() => {
      ChromeSamples.log('Text copied.');
```